src/test/java/academy/softserve/edu/tests/supervisor/CreateReportTest.java

```diff
@@ -0,0 +1,39 @@
+package academy.softserve.edu.tests.supervisor;
+
+import academy.softserve.edu.enums.Roles;
+import academy.softserve.edu.utils.TestRunner;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static academy.softserve.edu.asserts.AbstractElementAssert.assertThat;
+
+public class CreateReportTest extends TestRunner {
+
+    @BeforeMethod
+    public final void setUpTests() {
+
+        userInfoPage = logInPage.logInAs(Roles.SUPERVISOR);
+        itemManagementPage = userInfoPage.clickItemManagementTab();
+    }
+
+    @Test
```

Comment:
remove that, you click that link in the next test, if it's not there, it'll fail anyway